docs/en/reference/security.rst

```diff
@@ -90,7 +90,7 @@ Instead of using string concatenation to insert user-input into your SQL/DQL sta
 placeholders and then explain to the database driver which variable should be bound to
 which placeholder. Each database vendor supports different placeholder styles:
 
--  All PDO Drivers support positional (using question marks) and named placeholders (:param1, :foo, :bar).
+-  All drivers support positional (using question marks) and named placeholders (e.g. ``:param1``, ``:foo``).
```

Comment:
"All drivers support positional" — this is not true. The original statement was correct.